### ISSUE TITLE
image layers

### ISSUE THREAD

**Issue by fkarasek** [google-research/fixmatch#27]
Hello Google-Research,

Amazing work! I'm just wondering if you know which layer contains the image layers or representations of images after the model is fully trained. 

Thanks again for posting this. It has been very useful.

**Comment by david-berthelot** [comment]
There's nothing specific in our neural network, it's a standard ResNet. Generally speaking I believe the accepted practice in ML is to use as a representation the inputs to the layers that project to logits. So the inputs to the last one. But let's be clear it's not the scope of the paper and we make no such claims.

**Comment by fkarasek** [comment]
I totally understand! It most certainly isn't in the scope of the paper. But I'm still eager to learn and pick the minds of the experts. Thanks very much for this answer.
